test/test_torch.py

```diff
@@ -2750,6 +2764,11 @@ def test_tensor_factory(self):
                 a[0] = 7.
                 self.assertEqual(5., res1[0].item())
 
+        # test boolean tensor
+        a = torch.tensor([True, True, False, True, True], dtype=torch.bool)
+        b = torch.tensor([-1, -1.1, 0, 1, 1.1], dtype=torch.bool)
```

Comment:
Not for this PR, but we should probably warn in this case... it seems unlikely that the user actually intended for this to happen

Reply:
Added to my TODO list